README.md

```diff
@@ -1 +1,53 @@
-# portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+-Portfolio setup for mobile
```

Comment:
[OPTIONAL] Kindly give a detail description of what your project is about. This will help others understand the project before even delving into it.

Reply:
@jerryowusu thanks for the very detailed and  informative feedback